### ISSUE TITLE
Match properties does not work with text overides [Bug]: 

### ISSUE THREAD

**Issue by TBAARCH** [pyrevitlabs/pyRevit#2260]
### ✈ Pre-Flight checks

- [X] I **don't have SentinelOne** antivirus installed (see above for the solution)
- [X] I have **searched in the issues** (open and closed) but couldn't find a similar issue
- [X] I have **searched in the [pyRevit Forum](https://discourse.pyrevitlabs.io)** for similar issues
- [X] I already **followed the [installation troubleshooting guide](https://pyrevitlabs.notion.site/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7) thoroughly**
- [X] I am using the **latest pyRevit Version**

### 🐞 Describe the bug

Hi there! Love what you are doing. In previous versions of pyRevit, the match tool would work with text overrides in dimensions. That was so effective for us, we started relying on this tool to create documentation for wall finishes in architectural drawings by simply creating a text override in one dimension (i.e. PT-1 for a paint finish) and then match it to other strings to create a finish drawing. It was REALLY efficient. In the current of version of pyRevit, this finction is not working although the option is checked in the Match Options dialogue.
![image](https://github.com/pyrevitlabs/pyRevit/assets/170007734/b9e6edf0-6155-410a-b777-70132c70fa5f)

Is it possible to have this fixed?
Thanks again for the great work!

### ⌨ Error/Debug Message

```shell
Not Applicable
```

### ♻️ To Reproduce

NOt Applicable. Function just does not work in current version.

### ⏲️ Expected behavior

Match tool would change text overrides in subsequently picked strings.

### 🖥️ Hardware and Software Setup (please complete the following information)

```shell
Not applicable.
```

**Comment by MohamedAsli** [comment]
Hi @TBAARCH,
If it's not too late :) 
Can you plase provide an example, and specify on which version it broke/is breaking? I couldn’t reproduce it on my side.
I’m currently on the WIP version, and it’s working just fine and effectively matching all fields, including Replaced Value.

**Comment by MohamedAsli** [comment]
@jmcouffin is it okay to manually Stale-label issues ?

**Comment by jmcouffin** [comment]
yes, but for this one, since you asked for more info, there is a label for that.
